src/EnvJson.php

```
@@ -60,85 +66,140 @@
 
-        if ($envDistJsonFile) {
-            return json_decode(file_get_contents($envDistJsonFile), true, 512, JSON_THROW_ON_ERROR); // @phpstan-ignore-line
+        if ($envDistJsonFile !== false) {
+            $contents = file_get_contents($envDistJsonFile);
+            if ($contents === false) {
+                throw new RuntimeException("Failed to read env file: {$envDistJsonFile}"); // @codeCoverageIgnore
+            }
+
+            $decoded = json_decode($contents, true, 512, JSON_THROW_ON_ERROR);
+            if (! is_array($decoded)) {
+                throw RuntimeException("Invalid JSON format in env file: {$envDistJsonFile}. Expected array."); // @codeCoverageIgnore
```

PHPStan (2.0.3)

Function RuntimeException not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)